OlympGuide.Application/Features/SportField/NoSportFieldFoundException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OlympGuide.Application.Features.SportField
+{
+    public class NoSportFieldFoundException: Exception
+    {
+        public NoSportFieldFoundException() : base("No sport field was found.")
```

Comment:
`public NoSportFieldFoundExcetption(Guid id) : base("String.Format("Sport field with id: %s was not found", id.ToString())`